src/gluonts/model/forecast.py

```diff
@@ -188,7 +188,32 @@ def as_json_dict(self, config: "Config") -> dict:
         if OutputType.samples in config.output_types:
             result["samples"] = []
 
-        return result
+        return Forecast.jsonify_floats(result)
+
+    @staticmethod
+    def jsonify_floats(json_object):
```

Comment:
I think this is not the right place for the method. Maybe, we could add it to some `util` module. And maybe it's also better to move it to `shell`.

Reply:
Done.